app/connectors/itsastatus/ITSAStatusUpdateConnector.scala

```diff
@@ -78,11 +78,53 @@ class ITSAStatusUpdateConnector @Inject()(val httpClient: HttpClientV2, val appC
                     .map(failure => s"code: ${failure.code}, reason: ${failure.reason}")
                     .getOrElse("unknown reason")
 
-                log.error(s"response status: ${response.status}, message: $message")
+                logger.error(s"response status: ${response.status}, message: $message")
                 valid
               }
             )
         }
       }
   }
+
+  def makeMultipleItsaStatusUpdateRequests(
```

Comment:
The Sign up journey doesn't appear to be using this

Reply:
removed